repo/content/content_manager_test.go

```diff
@@ -1804,17 +1804,19 @@ func (s *contentManagerSuite) TestContentWriteAliasing(t *testing.T) {
 		"k",
-		NoCompression)
+		compression.HeaderZstdFastest)
```

Comment:
The existing test was verifying that even without any compression set, the default policy is to compress using zstd-fastest. IS that correct ?

If so, then would it be better to add a new test for your policy driven metadata compression settings ?

Reply:
I've added a separate test for NoCompression